docs: Debug documentation and usages (#267)
valkey-io/valkey-glide-docs: src/content/docs/concepts/client-features/custom-commands.mdx

In PHP, this method is called `rawcommand()` instead of `custom_command`.
:::

> To see how to use `custom_command`, refer to the [Execute Custom Commands](/how-to/execute-custom-commands/) guide.

Reviewer: ```suggestion
For `custom_command` examples, refer to the [Execute Custom Commands](/how-to/execute-custom-commands/) guide.
```

Non-native speakers sometimes have a hard time with phrasing like that.
